contracts/src/tests/fork_tests/test_ekubo.cairo

```diff
@@ -250,6 +250,24 @@ fn test_launch_meme() {
         );
 }
 
+#[test]
+#[fork("Mainnet")]
+fn test_transfer_ekuboLP_position() {
+    let owner = snforge_std::test_address();
+    let (quote, quote_address) = deploy_eth_with_owner(owner);
+    let starting_tick = i129 { sign: true, mag: 4600158 }; // 0.01ETH/MEME
+    let (memecoin_address, id, position) = launch_memecoin_on_ekubo(
+        quote_address, 0xc49ba5e353f7d00000000000000000, 5982, starting_tick, 88719042
+    );
+
+    // Execute the transfer of position
+    let ekubo_launcher = IEkuboLauncherDispatcher { contract_address: EKUBO_LAUNCHER_ADDRESS() };
+    ekubo_launcher.transfer_position_ownership(id, ALICE());
+
+    // Check that LP transfer to Alice is successful
```

Comment:
should also check that the remaning positions for `owner=snforge_std::test_address()` is now empty

and that when you query the `liquidity_position_details(id)` the owner returned is now ALICE()

Reply:
done!